Amazon.Ion.ObjectMapper/IonPrimitiveSerializers.cs

```diff
@@ -65,7 +65,7 @@ public class IonLongSerializer : IonSerializer<long>
 
         public long Deserialize(IIonReader reader)
         {
-            return reader.IntValue();
+            return reader.LongValue();
```

Comment:
Related to this change? Seems like it should be a separate PR if it was discovered here.